client/src/components/QuestionsAndAnswers/QuestionsList/QuestionsList.jsx

```diff
@@ -2,12 +2,32 @@ import React from 'react';
 import './QuestionsList.scss';
 import QuestionListItem from '../QuestionListItem/QuestionListItem.jsx';
 
-const QuestionsList = () => {
+const QuestionsList = ({
+  questionsAndAnswers,
+  onQuestionHelpulButtonClick,
+  onAnswerHelpulButtonClick,
+  setShowAddAnswerModal
+}) => {
   return (
     <>
-      <ul>
-        <QuestionListItem />
-        <QuestionListItem />
+      <ul className="question-list">
+        {questionsAndAnswers.map(qa => {
+          console.log('Question List QA', qa);
+          return (
+            <QuestionListItem
+              key={qa.question_id}
```

Comment:
The code might be easier to read if you just past the entire qa as an object into the sub-component. So you don't have to keep track of all the variables.

Reply:
I see your point and I agree with it but I prefer to exposed all the props to the components consumer and this way we can be sure that all the required data is passed in.